### PR TITLE
new extension for nexus identity feature

### PR DESCRIPTION
**Pull request by jtamma**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [X] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally? (`pip install wheel==0.30.0` required)
- [X] My extension version conforms to the [Extension version schema](https://github.com/Azure/azure-cli/blob/release/doc/extensions/versioning_guidelines.md)

For new extensions:

- [X] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your pull request is merged into main branch, a new pull request will be created to update `src/index.json` automatically.  
You only need to update the version information in file setup.py and historical information in file HISTORY.rst in your PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
Thank you for your contribution! We will review the pull request and get back to you soon.

File: src/nexusidentity/setup.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+from setuptools import setup, find_packages
+try:
+    from azure_bdist_wheel import cmdclass
+except ImportError:
+    from distutils import log as logger
+    logger.warn("Wheel is not available, disabling bdist_wheel hook")
+
+# TODO: Confirm this is the right version number you want and it matches your
+# HISTORY.rst entry.
+VERSION = '0.1.0'
```
Comment (AllyW) [inline src/nexusidentity/setup.py:19]: please update version as recommended above. For more cli extension schema, please refer to this doc: https://github.com/Azure/azure-cli/blob/dev/doc/extensions/versioning_guidelines.md

**Comment by jtamma** [comment]
@yonzhan @AllyW @jsntcy @kairu-ms 
Please review

**Comment by jtamma** [comment]
@AllyW 
Can you please help to review this PR. Also can you help me with the milestone dates on which the review will take place.

**Review by AllyW — APPROVED** [review]

**Comment by AllyW** [comment]
> @AllyW Can you please help to review this PR. Also can you help me with the milestone dates on which the review will take place.

@jtamma it should be released in following several hours.

**Pull request merged**